app/scripts/models/reliers/oauth.js

```diff
@@ -28,6 +28,8 @@ define(function (require, exports, module) {
   var SIGNIN_SIGNUP_QUERY_PARAM_SCHEMA = {
     access_type: Vat.accessType().renameTo('accessType'),
     client_id: Vat.clientId().required().renameTo('clientId'),
+    code_challenge: Vat.string().min(43).max(128),
```

Comment:
For consistency with other model items, can you apply `renameTo` and use camelCase?